jubatus/core/driver/test_util.hpp

```diff
@@ -51,6 +53,21 @@ jubatus::util::lang::shared_ptr<jubatus::core::fv_converter::datum_to_fv_convert
   return converter;
 }
 
+jubatus::core::fv_converter::datum generate_random_datum(
+    jubatus::util::math::random::mtrand& rand,
+    int datum_length) {
+  jubatus::core::fv_converter::datum d;
+  std::string value;
+  value.resize(datum_length * 2 + 1);
```

Comment:
`resize` fills `'\0'`. It is not recommended to set '\0' in std::string when the string is used as string.